Documentation/gettingstarted/egress-gateway.rst

```diff
@@ -130,6 +130,9 @@ on the interface they are assigned to, otherwise the return traffic won't be abl
 
 .. literalinclude:: ../../examples/kubernetes-egress-gateway/egress-ip-deployment.yaml
 
+Please keep in mind that the deployment is used only as an example how to set the egress IP, and it is not
+suitable for a production use case.
```

Comment:
I understand the motivation for this note, but I'm not sure how an external reader would read it.

Say I'm looking for an example file on which I want to base my own deployment. Does this note mean that the feature itself is unstable and I should not use it? Or that just this example deployment is not for production? In that case, why should it not be reused - what is missing to make it production ready? If my own deployment file is similar to this example, does this mean I made a mistake somewhere?

I don't have an ideal wording for the notes, but here are some suggestions:
- `the deployment` -> `the example deployment provided above`, or something like this, to make sure we understand what deployment we're talking about
- Maybe explain briefly why it is not suitable for a production use case?